training_pipeline/pipeline/components/HFPusher/model_card_template.md

```diff
@@ -0,0 +1,27 @@
+---
+# For reference on model card metadata, see: https://github.com/huggingface/hub-docs/blob/main/modelcard.md?plain=1
```

Comment:
Does this link also show how we can specify the formatting like:

```
This is a {{ task | default("segmentation")}} model trained using Vertex AI.
```

Reply:
it just shows the keys, so nope.